src/ImageSharp/Formats/Bmp/BmpInfoHeader.cs

```diff
@@ -6,90 +6,152 @@ namespace SixLabors.ImageSharp.Formats.Bmp
     /// This block of bytes tells the application detailed information
     /// about the image, which will be used to display the image on
     /// the screen.
-    /// <see href="https://en.wikipedia.org/wiki/BMP_file_format"/>
+    /// <seealso href="https://en.wikipedia.org/wiki/BMP_file_format">See this Wikipedia link for more information.</seealso>
     /// </summary>
     internal sealed class BmpInfoHeader
     {
         /// <summary>
-        /// Defines the size of the BITMAPINFOHEADER data structure in the bitmap file.
+        /// If this field is diferent of <see cref="BmpOS2Compression.RGB"/>, an IBM OS/2 <c>OS2InfoHeaderV2</c> DIB header
+        /// is being used insted of the Microsoft Windows <see cref="WinInfoHeaderV5"/> DIB header.
+        /// See <see cref="OS2InfoHeaderV2.Compression"/>.
         /// </summary>
-        public const int BitmapInfoHeaderSize = 40;
+        public BmpOS2Compression OS2Compression = BmpOS2Compression.RGB;
```

Comment:
This line is breaking the build. It needs to be a property.

Reply:
Hi,

My latested GIT version submited should fix this.